_posts/2016-03-07-2016-forman-survey-analysis.md

```diff
@@ -0,0 +1,342 @@
+cut'n'paste the entire result set, but we did get some themes. Out of the 17
+responses to this question:
+
+* 11 were generally positive about the support/docs (65%)
+* 7 requested more diagrams or how-to guides (potentially including screencasts) (41%)
+* 3 specifically requested improvements to plugin docs (18%)
+
+It's good to see two-thirds of the responses are positive, as a lot of effort
+goes into the manual and support channels. I still plan to produce my
+"beginners guide" screencast series "soon(tm)". Plugins, as ever, remain the
+responsibility of the authors, but the development community tries to help out
+where we can. Writing docs is also a great way to get [started with
+contributing][contribute], if you're so inclined!
+
+## <a name="foreman-features"></a>Specific Foreman Features
+
+Now we move onto the detail in some of Foreman's features:
+
+![Compute Resources](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/compute-resources.png)
+
+This graph is fascinating for quite a few reasons. Firstly, in a change from
+last year, we explicitly included "Physical hardware" as an option - although
+the question is about Compute Resources, in reality it should have been "What
+do you provision on with Foreman?". By including this separately from "None",
+we get that nearly two-thirds of the community stil manage physical hardware
+with Foreman. PXE is not dead yet, it seems!
+
+The other interesting figures are in the shifts from last year. EC2 and
+OpeStack both lose a few percentage points, while VMware and Docker (no
+surprise there) gain a few. The really big increases, though, are in Libvirt
+and oVirt, which given the Docker hype this year is really surprising to me.
+Other is also up substantially, perhaps in a nod to our plugin structure for
+Compute Resources.
+
+Overall, this graph nicely highlights the diversity of infrastructure that
+we're able to support, and cautions us about getting to narrow-minded with
+respect to a particular platform. We do well because of our neutrality.
+
+![Taxonomies](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/taxonomies.png)
+
+One of our more advanced features, Locations / Organisations are a scoping
+mechanism, useful if you have resources that should only matter to certains
+places or people (e.g. multi-tenant setups). Whats interesting here is that
+while the number of users not using Locs/Orgs is constant, many of those who
+used just one last year now use both (24% -> 33%).
+
+We actually asked three questions on this feature, as the developer community
+wanted to gather usecases for a potential refactoring of the feature, and
+wanted to be sure we didn't break it for anyone. We'll be able to say more on
+this once the refactoring is decided on (or not)
+
+This next graph is huge. Sorry ;)
+
+![Plugin Usage](/static/images/blog_images/2016-03-07-2016-forman-survey-analysis/plugin-usage.png)
```

Comment:
I'd remove the ones with 0% usage from the graph, these are clearly not important for many people - if Google forms cannot produce a graph with the full names, do like in the IRC pic, list the options